roles/edpm_update/molecule/default/verify.yml

```diff
@@ -0,0 +1,22 @@
+- name: Verify OVS update
+  hosts: all
+  vars:
+    test_helper_dir: "../../../../molecule/test-helpers"
+  tasks:
+    - name: Ensure openvswitch systemd services are in appropriate states after the update
```

Comment:
this is good; I think we will also need to verify that flows are not gone during this process; that the pid of the vswitchd process is the same; perhaps some other runtime checks to make sure it's not touched during package update, need to think what else we should check.